### PR TITLE
Expanding the list of available permissions under global security.

### PR DESCRIPTION
**Pull request by azafty468**
I've added 8 permissions typically available under global security.  I've also updated the test file however I am unable to run the tests.  When I run 'make run' I fail on step 23 with the following error:
`
Step 23/27 : COPY plugins $JENKINS_HOME/plugins/
COPY failed: stat /var/lib/docker/tmp/docker-builder898613473/plugins: no such file or directory
make: *** [build] Error 1
`
My additions are trivial so I believe there is a low chance of error.  Please let me know if there is anything I should do to complete testing.

### CODE REVIEW

**Comment by estute** [comment]
Ah @azafty468 concerning the error you mentioned. Prior to building the container, make sure that you run 'make plugins', which manages dependencies for the plugins and downloads them.

I usually do the following to get a fully functional jenkins container:
```
source local_env.sh
make requirements plugins build run
```
Let me know if that works for you.

**Comment by estute** [comment]
Concerning the first bok-choy failure on Travis (<e2e.test_ghprb_plugin_configuration.TestGHPRBConfiguration testMethod=test_ghprb_config>), this test is unfortunately fairly flaky and can be ignored, seeing that you are not touching anything GHPRB related. The main Jenkins configuration page is getting quite large and often it can be tricky to scroll around on.

File: test_data/security.yml [hunk 1]
```diff
@@ -16,6 +16,11 @@ SECURITY_GROUPS:
         - 'anonymous'
     - NAME: 'jenkins admins'
       PERMISSIONS:
```
Comment (estute) [inline test_data/security.yml]: No commas needed here

**Comment by HassanJaveed84** [comment]
@estute can you please review this ?

**Pull request merged**